lib/lita/handlers/reviewme.rb

```diff
@@ -123,6 +123,14 @@ def github_access_token
       def chat_mention(reviewer, url)
         "#{reviewer}: :eyes: #{url}"
       end
+
+      def get_room(response)
+        response.message.source.room
```

Comment:
Hmm, room appears to be delegated, so I can violate demeter slightly less by removing `.source` here https://github.com/litaio/lita/blob/94ad234c9737d658d1b0d8d7829530e4b378dc62/lib/lita/message.rb#L30